src/App.jsx

```diff
@@ -1,15 +1,18 @@
 import React from 'react';
+import { Sum } from './components/Sum';
 import './App.scss';
 
 const App = () => (
   <>
```

Comment:
you don't need a fragment here

Reply:
I just learned this in a lecture)
Ty :D